src/components/HookForm/index.jsx

```diff
@@ -0,0 +1,464 @@
+                    secondary={hook.status.lastFire.taskId}
+                  />
+                  <LinkIcon />
+                </ListItem>
+              ) : (
+                <ListItem>
+                  <ListItemText
+                    primary="Last Fired Result"
+                    secondary={
+                      lastFireTypeName === HOOKS_LAST_FIRE_TYPE.NO_FIRE ? (
+                        'n/a'
+                      ) : (
+                        <pre>
+                          {JSON.stringify(hook.status.lastFire.error, null, 2)}
+                        </pre>
+                      )
+                    }
+                  />
+                </ListItem>
+              )}
+              <ListItem>
+                <ListItemText
+                  primary="Next Scheduled Fire"
+                  secondary={
+                    hook.status.nextScheduledDate ? (
+                      <DateDistance from={hook.status.nextScheduledDate} />
+                    ) : (
+                      'n/a'
+                    )
+                  }
+                />
+              </ListItem>
+            </Fragment>
+          )}
+          <List subheader={<ListSubheader>Schedule</ListSubheader>}>
+            <ListItem>
+              <ListItemText
+                primary={
+                  <TextField
+                    name="scheduleTextField"
+                    placeholder="* * * * * *"
+                    fullWidth
+                    onChange={this.handleInputChange}
+                    value={scheduleTextField}
+                  />
+                }
+              />
+              <IconButton
+                className={classes.iconButton}
+                onClick={this.handleNewCronJob}>
+                <PlusIcon />
+              </IconButton>
+            </ListItem>
+            {schedule.map(cronJob => (
+              <ListItem key={cronJob}>
+                <ListItemText primary={<code>{cronJob}</code>} />
+                <IconButton
+                  className={classes.iconButton}
+                  name={cronJob}
+                  onClick={this.handleDeleteCronJob}>
+                  <DeleteIcon />
+                </IconButton>
+              </ListItem>
+            ))}
+          </List>
+          <List subheader={<ListSubheader>Task Template *</ListSubheader>}>
+            <ListItem className={classes.editorListItem}>
+              <Typography variant="caption">
+                <span>
+                  When the hook fires, this template is rendered with{' '}
+                  <a
+                    href="https://taskcluster.github.io/json-e/"
+                    target="_blank"
+                    rel="noopener noreferrer">
+                    JSON-e
+                  </a>{' '}
+                  to create the the task definition. See{' '}
+                  <a
+                    href="https://docs.taskcluster.net/reference/core/taskcluster-hooks/docs/firing-hooks"
```

Comment:
Can we bring in `taskcluster-lib-urls` for this?

Reply:
Sure, but I think the generated link won't work right away.